mnist/requirements.txt

```diff
@@ -1,2 +1,2 @@
 torch
-torchvision==0.20.0
+torchvision==0.21.0
```

Comment:
I think these examples should work on the latest torch and torchvision packages unless they deliberately test the deprecated API and soon will be dropped. As such we need to drop version pin entirely instead. That was done in the recently merged #1334 for this example. I suggest to close this PR. Otherwise, please, join discussion in issue #1329.